SAP_DEVELOPMENT_GUIDE.md

```
@@ -0,0 +1,999 @@
+# Sync single skill
+./scripts/generate-plugin-manifests.sh sap-cap-capire
+```
+
+**When to Run**:
+- After adding new skills
+- Before releases
+- After updating skill YAML frontmatter
+- When version changes in marketplace.json
+
+**Output**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)